.github/workflows/ci.yml

```diff
@@ -395,6 +395,7 @@ jobs:
         shell: bash -l {0}
     env:
       MODIN_STORAGE_FORMAT: "hdk"
+      PYTHONPATH: "."
```

Comment:
Why do we need this?

Reply:
Without this, the benchmarks fail on the modin import.

Reply:
Do we have the fail locally? If so, we should probably make a respective change in the code itself.

Reply:
Yes, all 3 tests fail for me with the same error: ModuleNotFoundError: No module named 'modin'

Reply:
I've removed the PYTHONPATH env variable and added corresponding changes to the code.